caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters to run online motion correction followed by online CNMF 
+
+            estimates: Estimates, optional
+                Estimates object to load an existing model
+
+            path: str, optional
+                path to a saved OnACID model hdf5 file on disk
```

Comment:
Might be worth linking to a doc (although I don't think we have this yet) on what format this model is in, how to make one, etc etc.. I'd note that as a TODO here

Reply:
The docs and nb examples for how to save and load models are very sparse, this is a general thing that needs reworking.

Reply:
I made #1141 for this